patches/chrome-browser-ui-webui-chrome_web_ui_controller_factory.cc.patch

```diff
@@ -19,3 +19,11 @@ index be8b073e7bd2778e03d99f953ffae13d90b3b151..b7308cd52bcbb50d9ed6f8e95f3c47cc
  }
  
  // static
+@@ -788,6 +789,7 @@ bool ChromeWebUIControllerFactory::IsWebUIAllowedToMakeNetworkRequests(
+   // If you are adding a new host to this list, please file a corresponding bug
+   // to track its removal. See https://crbug.com/829412 for the metabug.
+   return
++      origin.host() == "bravesynclib" ||
```

Comment:
this shouldn't be needed, we aren't making network calls from the UI, only from the background page

Reply:
Yes, sure. I will remove it once complete move to background page.